projects/10up-theme/functions.php

```diff
@@ -14,7 +14,11 @@
 define( 'TENUP_THEME_INC', TENUP_THEME_PATH . 'includes/' );
 define( 'TENUP_THEME_BLOCK_DIR', TENUP_THEME_INC . 'blocks/' );
 
-if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG && file_exists( __DIR__ . '/dist/fast-refresh.php' ) ) {
+$is_local_env = in_array( wp_get_environment_type(), [ 'local', 'development' ], true );
+$is_local_url = strpos( home_url(), '.test' ) || strpos( home_url(), '.local' );
```

Comment:
If you only support WP 5.9 or later (or PHP 8.0 or later) then you can use `str_ends_with()` to be more precise -- WP polyfills the function in 5.9 or later.

If you need to support WordPress 5.8 and earlier then cool story, please click resolve conversation. :)

Reply:
We don't have clear minimum requirements but I know some projects take longer to update WP so I'm going to be conservative and don't use `str_ends_with`. Thanks for the tip!